style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1c1a19;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+  
+li {
+  list-style: none;
+  flex-direction: row;
+}
+ 
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+}
+  
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: white;
+}
+  
+header a {
+  color: white;
+  text-decoration: none;
+  font-size: 17px;
+}
+  
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  font-family: 'Roboto', sans-serif;
+  background: #3c3a39;
+  align-items: center;
+}
+
+.hero-wrapper {
+  position:absolute;
```

Comment:
- [ ] You should use flexbox for layout. Please change it. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox%20in%20all%202%20sections.)